crates/rust-analyzer/src/config.rs

```diff
@@ -456,7 +456,8 @@ config_data! {
         /// Additional arguments to `rustfmt`.
         rustfmt_extraArgs: Vec<String>               = "[]",
         /// Advanced option, fully override the command rust-analyzer uses for
-        /// formatting.
+        /// formatting. This should be the equivalent of `rustfmt` here, and
+        /// not that of `cargo fmt`.
```

Comment:
It might be worth making explicit that the command should read code to format from stdin and write formatted code back to stdout.